### PR TITLE
Created Custom FormField for required! Cleaned up stories

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?

Adds a story around how to create a custom Form Field component that will allow the user to customize the type of label they want. Also separated stories to individual files to clean up code  

#### Where should the reviewer start?

FormField/stories/RequiredLabel.js

#### What testing has been done on this PR?

auto

#### How should this be manually tested?
Storybook
#### What are the relevant issues?
#3140 
#### Do the grommet docs need to be updated?
no
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: src/js/components/FormField/stories/RequiredLabel.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const LabelFormField = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large">
+      <Form>
+        <FormFieldLabel name="firstName" label="FirstName" required />
+        <FormFieldLabel name="LastName" label="LastName" required />
+        <FormFieldLabel name="email" label="Email" />
+        <Button type="submit" label="Submit" primary />
+        <Text margin={{ left: 'small' }} size="small" color="red">
+          {' '}
```
Comment (ShimiSun) [inline src/js/components/FormField/stories/RequiredLabel.js]: Can we remove this line of code?

File: src/js/components/FormField/stories/RequiredLabel.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+const LabelFormField = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large">
+      <Form>
+        <FormFieldLabel name="firstName" label="FirstName" required />
+        <FormFieldLabel name="LastName" label="LastName" required />
+        <FormFieldLabel name="email" label="Email" />
+        <Button type="submit" label="Submit" primary />
+        <Text margin={{ left: 'small' }} size="small" color="red">
```
Comment (ShimiSun) [inline src/js/components/FormField/stories/RequiredLabel.js]: I think we would want to use grommet colors here, what about `status-error`? https://v2.grommet.io/color

File: src/js/components/FormField/stories/RequiredLabel.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+const FormFieldLabel = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={
+        required ? (
+          <Box direction="row">
+            <Text>{label}</Text>
+            <Text color="red">*</Text>
```
Comment (ShimiSun) [inline src/js/components/FormField/stories/RequiredLabel.js]: I think we would want to use grommet colors here, what about `status-error`? https://v2.grommet.io/color

File: src/js/components/FormField/stories/RequiredLabel.js (hunk 2 above)
Comment (britt6612) [reply src/js/components/FormField/stories/RequiredLabel.js]: Of course I can change this!

**Comment by ShimiSun** [comment]
Looks greThanks for the contribution! 🎉

**Pull request merged**